slovenian/rumena/rumena.md

```diff
@@ -1 +1,8 @@
 Znašel si se v rumeni sobi z dvemi vrati.
+In pred tabo stojijo štirje psihopati,
+Dva imata v rokah nož, druga dva sekiro,
+A ti imaš pred sabo sledečo izbiro: 
+
+[Baci bombu, goni bandu](baci-bombu/baci-bombu.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.